backend/tracim_backend/templates/mail/content_update_body_html.mak

```diff
@@ -1,4 +1,6 @@
-## -*- coding: utf-8 -*-
+<!DOCTYPE html>
+<html lang="${lang}">
+<body>
 	<!--FIXME After discuss with Damien: dont add intelligent code in template -->
```

Comment:
This FIXME still required ?

Reply:
Yes, i just moved fixme in code instead of rendered template.